src/epub/css/local.css

```diff
@@ -5,13 +5,33 @@
 	font-style: italic;
 }
 
-[epub|type~="z3998:letter"] p{
+#combat [epub|type~="z3998:letter"] p{
 	font-variant: small-caps;
 	text-align: initial;
 	text-indent: 0;
 	margin-bottom: 1em;
 }
 
+[epub|type~="z3998:letter"] header{
+	text-align: right;
+}
+
+footer{
+	margin-top: 1em;
+	text-align: right;
+}
```

Comment:
footer used to have `font-variant: normal`, is removing it correct?

Reply:
fixed